pkg/scheduler/internal/podinfo/podinfo_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package util
+package podinfo
```

Comment:
why are we going to move this into internal util pkg? It maybe useful for customized scheduler by framework.

Reply:
PR #76990 has discussion regarding moving this into an internal package

Reply:
@k82cn Can you confirm if we can move it into an internal package?